### PR TITLE
99.06% Classes

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Classes and some client rendering stuff

### CODE REVIEW

File: mappings/net/minecraft/util/math/noise/NoiseSampler.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_586 net/minecraft/util/math/noise/SimplexNoiseSampler
+CLASS net/minecraft/class_586 net/minecraft/util/math/noise/NoiseSampler
```
Comment (Grayray75) [inline mappings/net/minecraft/util/math/noise/NoiseSampler.mapping:1]: Why the rename?

**Review by Grayray75 — APPROVED** [review]

File: mappings/net/minecraft/util/math/noise/NoiseSampler.mapping (hunk 1 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/util/math/noise/NoiseSampler.mapping:1]: It doesn’t generate simplex noise. It’s used to generate specific kinds of noise

File: mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping [hunk 2]
```diff
@@ -14,6 +14,7 @@ CLASS net/minecraft/class_703 net/minecraft/client/gui/widget/PagedEntryListWidg
 		ARG 2 visible
 	METHOD method_2581 setActive (Z)V
 		ARG 1 active
+	CLASS class_704 ButtonEntry
```
Comment (hYdos) [inline mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping:17]: this doesnt seem right

File: mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping [hunk 3]
```diff
@@ -81,3 +82,7 @@ CLASS net/minecraft/class_703 net/minecraft/client/gui/widget/PagedEntryListWidg
 			ARG 2 mouseX
 			ARG 3 mouseY
 			ARG 4 button
+	CLASS class_709 ListEntry
+	CLASS class_710 LabelSupplierEntry
+		FIELD field_2980 labelSupplier Lnet/minecraft/class_690$class_691;
```
Comment (hYdos) [inline mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping:87]: this is a bit conc

**Review by hYdos — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping:87]: what should it do
it supplies the label

**Review by arthurbambou — APPROVED** [review]

**Comment by BoogieMonster1O1** [comment]
webhook test

**Pull request merged**